src/Rating/Rating.test.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ApiCalls from '../ApiCalls.js';
+import Rating from './Rating.js';
+import MovieView from '../MovieView/MovieView.js';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { BrowserRouter } from 'react-router-dom';
+jest.mock('../ApiCalls');
+
+describe('Rating', () => {
```

Comment:
This is a lot of testing for a small functional component. Did we go a little overboard, or should some of these tests maybe live on another component?

Reply:
That's an excellent point, I think that the extra tests are necessary to cover all the user functionality but maybe some are considered integration tests & therefore they should live elsewhere?